gemrb/tests/core/System/Test_VFS.cpp

```diff
@@ -23,11 +23,33 @@
 
 #include "System/VFS.h"
 
+#define IS_CASE_INSENSITIVE (WIN32 || __APPLE__)
```

Comment:
I guess we will change this later in #1892, though it's just the test suite, so maybe we don't care. Just today I was talking to a macos user with build problems due to using a case sensitive FS.

Reply:
Agree. Just don't have any better concept to deal with that r/n, also since this appears to be even a matter of per-path settings such as on ext4.